tests/unit/test_collection.py

```diff
@@ -263,6 +267,59 @@ def test_build_docs_blob_no_readme(get_readme_doc_file):
         collection_loader._build_docs_blob()
 
 
+@pytest.fixture
+def temp_root():
+    try:
+        tmp = tempfile.mkdtemp()
+        yield tmp
+    finally:
+        shutil.rmtree(tmp)
+
+
+EX_ENV_YAML = """version: 1
+dependencies:
+  python:
+    - somepkg==1.3
+    - otherpkg>=3.0
+  files:
+    - "/usr/bin/oc"
+    - "/usr/lib/libssl.so.1"
```

Comment:
I'm sorry about this schema, we've done a bad job of communicating the large amount of changes over to the issues you had open.

If you look at the README.md in https://github.com/ansible/ansible-builder, you can see the most up-to-date spec of how we will read data from collections. `files:` was never a part of what was actually being implemented, and there are other details about how the python/system deps are specified too.